### DISCUSSION TITLE
PDF sending via Email

### DISCUSSION THREAD

**Discussion by runesatsdev** [community/community#49496] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi 
I have a problem with pdf file send in my website.
If you have a experience with this problem then please help me

**Comment by 0xNevo** [comment, marked as answer]
Hi, It's simple.
using window.print function in javascript, can be implement easy.
Understand?

**Comment by marcleblack** [comment]
hello I have a problem to find smtp ci you had a solution let me know

**Reply by MichaelYani** [reply]
hello, you can find me in discord or skype.
I have solution for your worry.
discord: discord.com/users/1083507132760080384   or !Feed#8262
skype: https://join.skype.com/invite/wKkrO4UHXqN5 or https://join.skype.com/invite/wKkrO4UHXqN5
